src/Arcus.Templates.AzureFunctions.Databricks.JobMetrics/Startup.cs

```diff
@@ -41,8 +42,10 @@ public override void Configure(IFunctionsHostBuilder builder)
             LoggerConfiguration logConfig = CreateLoggerConfiguration(builder);
             builder.Services.AddLogging(logging =>
             {
-                logging.AddSerilog(logConfig.CreateLogger(), dispose: true);
+                logging.RemoveMicrosoftApplicationInsightsLoggerProvider()
+                    .AddSerilog(logConfig.CreateLogger(), dispose: true);
```

Comment:
Hihi, same as before, aligning on the `.` would be better.

Reply:
oh that's what you were aiming for. got it!

Reply:
https://github.com/arcus-azure/arcus.templates/pull/610/commits/d6f805e01da7787cf6826579a07d63af4d60c3ee